WooCommerce/Classes/ViewRelated/Inbox/Inbox.swift

```diff
@@ -61,6 +90,12 @@ private extension Inbox {
                                                          comment: "Title displayed if there are no inbox notes in the inbox screen.")
         static let emptyStateMessage = NSLocalizedString("Come back soon for more tips and insights on growing your store",
                                                          comment: "Message displayed if there are no inbox notes to display in the inbox screen.")
+        static let dismissAllNotification = NSLocalizedString("Dismiss All",
+                                                              comment: "Dismiss All button in Inbox Notes for dismissing all the notes.")
+        static let dismissAlertTitle = NSLocalizedString("Dismiss all messages",
+                                                         comment: "Title of the alert for dismissing all the inbox notes.")
+        static let dismissAlertMessage = NSLocalizedString("Are you sure? Inbox messages will be dismissed forever.",
```

Comment:
super nit: might be more clear
```suggestion
        static let dismissAllNotesAlertTitle = NSLocalizedString("Dismiss all messages",
                                                         comment: "Title of the alert for dismissing all the inbox notes.")
        static let dismissAllNotesAlertMessage = NSLocalizedString("Are you sure? Inbox messages will be dismissed forever.",
```